lib/bundler/cli/gem.rb

```diff
@@ -97,10 +98,14 @@ def run
             "test/test_helper.rb.tt" => "test/test_helper.rb",
             "test/newgem_test.rb.tt" => "test/#{namespaced_path}_test.rb"
           )
+        when "test-unit"
+          templates.merge!(
```

Comment:
Having `test/test_helper.rb` is better to share common helper codes.
See also: https://github.com/bundler/bundler/pull/5521/files#diff-7675ffc4b1b026242f4099e54c032b7b

Reply:
Thank you for reviewing this, but I believe it is greater to merge #5521 than this PR because that was created by the maintainer of test-unit.  Could you rebase #5521 if the feature request is accepted?

Reply:
OK.